### ISSUE TITLE
S3 cross-region replication for COMPLIANCE-mode documents bucket

### ISSUE THREAD

**Issue by alsiola** [percy-main/website-v2#218]
## Finding

`infra/modules/documents-bucket/main.tf:66-75` configures Object Lock COMPLIANCE mode for a 7-year retention bucket (safeguarding documents). No cross-region replication is configured.

For a long-retention COMPLIANCE bucket, region loss = data loss. Object Lock prevents tamper but cannot survive an `eu-west-1` outage / data corruption event.

## Fix

```hcl
resource \"aws_s3_bucket_replication_configuration\" \"documents\" {
  role   = aws_iam_role.replication.arn
  bucket = aws_s3_bucket.documents.id

  rule {
    id     = \"replicate-all\"
    status = \"Enabled\"

    destination {
      bucket        = aws_s3_bucket.documents_replica.arn
      storage_class = \"GLACIER_IR\"  # cheaper for replica
    }
  }
}
```

Plus `ReplicationLatency` alarm (target: < 15 min).

## Uncertainty

Adds AWS cost (replication GB-months + Glacier IR storage). Decide based on safeguarding policy / legal advisor input — see also memory note about non-profit credits.

## Acceptance

- [ ] Decision recorded with treasurer / safeguarding officer
- [ ] If approved, CRR + ReplicationLatency alarm in place

**Comment by alsiola** [comment]
Skipped during bug-batch-1 work — adding cross-region replication for the COMPLIANCE-mode documents bucket needs an explicit cost+governance decision (treasurer, safeguarding officer) that I can't make from here.

Open questions:
1. Is the safeguarding policy comfortable with a single-region COMPLIANCE bucket as today, or does it require CRR for regulator-grade durability?
2. Budget: CRR adds replication GB-month + Glacier IR storage in the destination region. Is this within the AWS-credits envelope (see project_nonprofit_credits memory)?
3. If approved, what's the destination region (eu-west-1 / eu-central-1 / similar)?

Once decided, the implementation is straightforward (per the issue body). Adding the `question` label.

**Comment by alsiola** [comment]
Not needed for level of compliance
